Description: adjust the test assertion to the actual error message
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: not-needed
Last-Update: 2024-05-31
--- a/tests/querystring.rs
+++ b/tests/querystring.rs
@@ -32,7 +32,7 @@
     assert!(params.is_err());
     assert_eq!(
         params.err().unwrap().to_string(),
-        "failed with reason: missing field `msg`"
+        "missing field `msg`"
     );
 }
 
@@ -47,7 +47,7 @@
     assert!(params.is_err());
     assert_eq!(
         params.err().unwrap().to_string(),
-        "failed with reason: missing field `msg`"
+        "missing field `msg`"
     );
 }
 
